config/profile_awsbatch_auto.config

```
@@ -7,12 +7,13 @@ aws{
     volumes = [
       '/ebs-autoscale:/tmp'
     ]
+    maxSpotAttempts = 2
   }
   region = 'us-east-1'
 }
 
 
 process{
   executor = 'awsbatch'
-  queue = 'nextflow-batch-autoscale-queue'
+  queue = { task.attempt < 2 ? 'nextflow-batch-autoscale-queue' : 'nextflow-batch-priority-queue'}
```

I think we want to do this on the third try?
```suggestion
  queue = { task.attempt < 3 ? 'nextflow-batch-autoscale-queue' : 'nextflow-batch-priority-queue'}
```